stdlib/src/builtin/file.mojo

```diff
@@ -303,9 +303,9 @@ struct FileHandle:
         ```mojo
         import os
         var file = open("/tmp/example.txt", "r")
-        var list1 = file.read(8)
+        var list1 = file.read_bytes(8)
```

Comment:
Nice! Indeed that looks like a mistake in the docs to use `read` instead of `read_bytes()`